src/components/Calculator.js

```diff
@@ -1,60 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.PureComponent {
-  constructor(props) {
-    super(props);
+function Calculator() {
```

Comment:
**Optional**
- Well done refactoring the component into a functional component. However, it'd be best practice if you used the arrow function ES6 feature 🙏

Reply:
Thanks! Changed it.